ts/nni_manager/common/utils.ts

```diff
@@ -188,8 +184,6 @@ function cleanupUnitTest(): void {
     Container.restore(DataStore);
     Container.restore(Database);
     Container.restore(ExperimentManager);
-    const logLevel: string = parseArg(['--log_level', '-ll']);
-    setExperimentStartupInfo(true, 'unittest', 8080, 'unittest', undefined, logLevel);
```

Comment:
why we set experiment info here before?

Reply:
<img width="580" alt="image" src="https://user-images.githubusercontent.com/16907603/160280856-a50ae8f5-5d4d-4b69-9ccc-3b7d040ab916.png">

btw, do we need to care about this information? (https://www.npmjs.com/package/typescript-ioc#using-container-for-testing)